crates/networking/rpc/Cargo.toml

```diff
@@ -37,12 +38,18 @@ keccak-hash.workspace = true
 
 cfg-if.workspace = true
 
+# Based
+ssz_types = { version = "0.10", optional = true}
+tree_hash = { version = "0.9.1", optional = true}
+tree_hash_derive = { version = "0.9.1", optional = true}
+strum_macros = { version = "0.24", optional = true }
```

Comment:
Same here

Reply:
Not sure about these. They are temporary as we'll replace the encoding with RLP.

Reply:
Let's add a comment or an issue so we don't keep them after the change.

Reply:
Done here https://github.com/lambdaclass/ethrex/issues/2306.